### PR TITLE
Remove personal fork dependencies

### PR DESCRIPTION
**Pull request by JeffreyMFarley**
While working on a separate task to address deprecation warnings from React, I noticed that we were still depending on work Richard had done in his personal forks.

The strategy here:

1. Capture the current look and feel of the Tile Map
1. Port the Britechart changes to the CFPB organization [(pull request)](https://github.com/cfpb/britecharts/pull/1)
1. Port the [landing page Tile Map](https://github.com/cfpb/cfgov-refresh/tree/ccdb-dev/cfgov/unprocessed/apps/ccdb-landing-map/js) from CFGOV-refresh
1. Point to the `cfpb/britecharts` package and remove the dependency on the `cfpb-chart-builder` package
1. Get testing back up

## Additional Yaks Shaved

- Address [deprecating lifecycle functions](https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html)
- Address [asynchronous test warnings](https://fb.me/react-wrap-tests-with-act)

## Testing

- :100:

## Screenshots

ccdb-ui-no-chart-builder

## Todos

- [ ] Rebuild the map when the window resizes
- [ ] Get the ported tile map tested

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/TileMap/__tests__/TileMap.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        {
+          color: 'rgba(247, 248, 249, 0.5)',
+          from: 1,
+          name: '≥ 1',
+          to: 16435
+        },
+        {
+          color: 'rgba(212, 231, 230, 0.5)',
+          from: 16435,
+          name: '≥ 16435',
```
[the remark this answers is not in the document]
Comment (JeffreyMFarley) [reply src/TileMap/__tests__/TileMap.spec.jsx]: Good catch

File: src/TileChartMap.jsx [hunk 2]
```diff
@@ -50,30 +50,26 @@ export class TileChartMap extends React.Component {
...
 
     const mapElement = document.getElementById( 'tile-chart-map' )
+    const offsetWidth = mapElement ? mapElement.offsetWidth : 800;
+
     // eslint-disable-next-line no-mixed-operators
-    const width = mapElement.offsetWidth - mapElement.offsetWidth * 0.1
+    const width = offsetWidth - offsetWidth * 0.1
 
-    const legendTitle = componentProps.dataNormalization === GEO_NORM_NONE ?
+    const description = componentProps.dataNormalization === GEO_NORM_NONE ?
```
[the remark this answers is not in the document]
Comment (JeffreyMFarley) [reply src/TileChartMap.jsx]: Done

File: src/TileMap/index.js [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+      chart: {
+        styledMode: true
+      },
+      colors,
+      colorAxis: {
+        dataClasses: bins,
+        dataClassColor: 'category'
+      },
+      title: false,
+      description: description,
```
[the remark this answers is not in the document]
Comment (JeffreyMFarley) [reply src/TileMap/index.js]: Done

File: src/TileMap/index.js [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+  const min = 0;
+
+  // Early exit
+  if ( max === 0 ) return [];
+
+  const step = ( max - min ) / binCount;
+  const bins = [];
+
+  for ( let i = 0, curr = min; i < binCount; i++, curr += step ) {
+    const minValue =  parseFloat( curr.toFixed( 2 ) )
```
Comment (flacoman91) [inline src/TileMap/index.js]: getting an eslint nag here about multiple spaces

Comment (JeffreyMFarley) [reply src/TileMap/index.js]: This is what happens when I only run `npm test` vs `npm start`

**Pull request merged**